app/collections/get-favorites-name.cjsx

```diff
@@ -0,0 +1,5 @@
+# Get the name of the favorites collection for a project
```

Comment:
Yeah I initially had just 'favorites'. The api scoping on the uniqueness validation is not defined to the project, so there can only be one 'favorites'. This is a workaround